docker/Dockerfile

```diff
@@ -15,14 +15,10 @@ RUN git clone https://github.com/aerogear/aerogear.org aerogear
 WORKDIR /home/aerogear/aerogear
 
 ## Setup Project
-  RUN  /bin/bash -l -c "rvm install ruby 2.1.2"
+  RUN  /bin/bash -l -c "rvm install ruby 2.4.1"
  RUN  /bin/bash -l -c "rvm gemset create aerogear"
- RUN  /bin/bash -l -c "rvm use ruby-2.1.2@aerogear"
+ RUN  /bin/bash -l -c "rvm use ruby-2.4.1@aerogear"
  RUN  /bin/bash -l -c "gem install bundler"
-USER root
-RUN dnf install -y libcurl-devel libcurl
-USER aerogear
- 
  RUN  /bin/bash -l -c "bundle install --path vendor"
```

Comment:
rss_atom_posts: fetching http://dcp.jboss.org/v1/rest/feed/?sys_type=blogpost&tag=feed_name_corinnekrych&tag=feed_name_abstractj&tag=feed_name_cvasilak&tag=feed_name_qmx_blog&tag=feed_name_passos&tag=feed_name_edewit&tag=feed_name_lfryc_events&tag=feed_name_blog_of_thrones&tag=feed_name_mwessendorf&tag=feed_name_sebiblog&tag=feed_name_secondsun&tag=feed_name_krisborchers
jekyll 3.0.0 | Error:  undefined method `items' for nil:NilClass

Reply:
@secondsun